roles/fzf/handlers/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
I've tended to not add files or directories for Ansible features I'm not using.

I also tend to forget they exist though, so having them included is a good idea.

Just some thoughts, I've yet to figure out a convention for these myself!

Reply:
I'm 50/50 on this. I even had them removed, in an earlier commit.

But as I'll probably use this role as a template for others, I ended up leaving them in. I might end up removing them again, as I add other roles.